src/main/java/org/mapstruct/intellij/util/MapstructAnnotationUtils.java

```diff
@@ -479,4 +482,78 @@ private static Stream<PsiClass> findReferencedMappersOfMapperConfig(PsiAnnotatio
         return findReferencedMappers( mapperConfigAnnotation );
     }
 
+    @NotNull
+    public static ReportingPolicy getReportingPolicyFromMethode( @NotNull PsiMethod method,
```

Comment:
I would suggest to rename this to `getUnmappedTargetPolicy` and only pass the `mappingMethod`. We are using this method in only one place so we can do the fallback in the implementation of the method and the use of `unmappedTargetPolicy`.

Reply:
Renamed to getUnmappedTargetPolicy